include/seqan3/utility/bloom_filter/bloom_filter.hpp

```diff
@@ -0,0 +1,385 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2020, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2020, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+/*!\file
+ * \author Tobias Loka <tobias.loka AT hpi.de>
+ * \brief Provides seqan3::bloom_filter.
+ */
+
+#pragma once
+
+#include <seqan3/std/algorithm>
+#include <seqan3/std/bit>
+#include <seqan3/search/dream_index/interleaved_bloom_filter.hpp>
+
+#include <sdsl/bit_vectors.hpp>
+
+#include <seqan3/core/concept/cereal.hpp>
+#include <seqan3/core/detail/strong_type.hpp>
+
+namespace seqan3
+{
+
+/*!\brief The bloom filter. A data structure that efficiently answers set-membership queries.
```

Comment:
```suggestion
/*!\brief The Bloom Filter. A data structure that efficiently answers set-membership queries.
```

Can you check that it's always **B**loom **F**ilter and **I**nterleaved **B**loom **F**ilter. While the capitalization of filter is debatable, Bloom is the name of the inventor and is hence always in upper case.